src/tasks/tasks.service.ts

```diff
@@ -39,7 +42,7 @@ export class TasksService {
     id: string,
     status: TaskStatus,
     user: User,
-  ): Promise<Task> {
+  ): Promise<GetTaskResDto> {
```

Comment:
It doesn't make any difference to only change the return type of the function. Inside the function, a new object with the type of GetTaskResDto must be created and the entity fields must be copied into that object's fields.

Reply:
Just for the sake of using typescript and consistency